hr_personal_equipment_request/models/hr_employee.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    equipment_request_ids = fields.One2many(
+        comodel_name="hr.personal.equipment.request",
+        inverse_name="employee_id",
+    )
+
+    personal_equipment_ids = fields.One2many(
+        comodel_name="hr.personal.equipment",
+        inverse_name="employee_id",
+        domain=[("state", "not in", ["draft", "cancelled"])],
+    )
+
+    equipment_request_count = fields.Integer(
+        compute="_compute_equipment_request_count",
+    )
+
+    personal_equipment_count = fields.Integer(
+        compute="_compute_personal_equipment_count"
+    )
+
+    def _compute_equipment_request_count(self):
+        self.equipment_request_count = len(self.equipment_request_ids)
+
```

Comment:
Why not 
```suggestion
        for rec in self:
            rec.equipment_request_count = len(rec.equipment_request_ids)

```
? 
Doing this way if you ever have to show this in a list view, the count will not be correct